docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -854,6 +854,25 @@ Cython code.  Here is the list of currently supported directives:
     signature, which cannot otherwise be retrieved after
     compilation.  Default is False.
 
+``embedsignature.format`` (``c`` / ``python`` / ``clinic``)
```

Comment:
@da-woods I reworded the documentation as  per your comments. Note however that the argument clinic tool does not generate signatures, instead, you have to write them by hand following the proper format.